src/main/webapp/intrigue-api/lib/cache.js

```diff
@@ -142,37 +162,42 @@ const getSourceStatus = queryId => state => {
 }
 
 const getTypes = state => {
-  return state.types.toJSON()
+  return getLocatlState(state).types.toJSON()
 }
 
 const getQueryResponse = queryId => state => {
-  return state.sourceResponse.get(queryId).toJSON()
+  return getLocalState(state)
+    .sourceResponse.get(queryId)
+    .toJSON()
 }
 
 const getMetacards = state => {
-  return state.results.toJSON()
+  return getLocalState(state).results.toJSON()
 }
 
 const getMetacard = metacardId => state => {
-  const metacard = state.results.get(metacardId)
+  const metacard = getLocalState(state).results.get(metacardId)
```

Comment:
What is this get local state pattern?

Reply:
We were just trying to group the query reducers under a unique key. 
The get local state is just pulling that key from state